### PR TITLE
added route for confirmation page and fixed redirection for signIn

### PR DESCRIPTION
**Pull request by n4dz**
I added the route for the confirmation page and the sign in page.
I also fixed redirection for sign In in the header.
(I know sign in is a stretch, but we now have the route if we need it)

### CODE REVIEW

File: client/src/components/App.js [hunk 1]
```diff
@@ -32,6 +35,12 @@ function App() {
         <Route path="/cart">
           <Cart />
         </Route>
+        <Route path="/confirmation">
```
Comment (carldumas) [inline client/src/components/App.js]: Hi @n4dz, 
Can we make the redirect point to 'order-confirmation' instead
I would also adjust the other one to be consistent.

Comment (n4dz) [reply client/src/components/App.js]: Hi @carldumas,
I made the changes 👍

File: client/src/components/App.js [hunk 2]
```diff
@@ -6,7 +6,10 @@ import Home from './home-page/Home';
 import Cart from './cart-page/Cart';
 import Item from './item-page/Item';
 import Error from './error-page/Error';
+import SignIn from './signin-page/SignIn';
+import OrderConfirmation from './oder-confirmation-page/OrderConfirmation';
```
Comment (carldumas) [inline client/src/components/App.js]: @n4dz 
Look like there is a little typo. ('./oder-...') We should read './order-...'
Please fix file name also

Comment (n4dz) [reply client/src/components/App.js]: Thanks, I changed it 👍

**Review by garorueda99 — APPROVED** [review]

**Pull request merged**